### ISSUE TITLE
Add constants for build status

### ISSUE THREAD

**Issue by strarsis** [RyanTheAllmighty/Docker-Hub-API#8]
Adding constants for the build status codes to docker-hub-api 
would make code more self-documentary:

```
[...]
if(build.status == dockerHubAPI.BUILD_STATUS_SUCCEEDED) {
[...]
if(build.status == dockerHubAPI.BUILD_STATUS_FAILED) {
[...]
```

**Comment by RyanTheAllmighty** [comment]
Yeah there are some changes I want to make for version 1.0.0 as I'm not entirely happy with the way things are at the moment with this package. While it works, it needs some work done and this is definitely a step towards a better library.

**Comment by strarsis** [comment]
There are 2 more status codes:
0 for queued
3 for building
